### PR TITLE
Add namespace inode numbers to process events

### PR DESCRIPTION
**Pull request by haesbaert**
This adds the inode numbers of 4 namespaces to both kprobe and ebpf backends:
 - uts
 - ipc
 - mnt
 - net

The interesting ones we are missing are pid and cgroups, the former is a bit
complex to fetch from kprobes and the later is not present in all kernels. Since
we don't have use for them at the moment, don't bother trying to include them.

The only real needed one at the moment is mnt so that auditbeat can check if it
is in the same mnt namespace as the process.

net offset location changed twice, so we need two alternatives instead of one,
but the existing infrastructure could not cope with two alternatives, so kill
btf_alternative_name() and loop over all possible alternatives to support it.

Note that if we fail to parse the sproc namespace entry, maybe cause the process
just vanished, we store zero which is an invalid inode number, the race recovers
as we only parse /sproc with the rings opened, meaning we will likely see the
exit event, which has the correct inodes.

### CODE REVIEW

File: btfhub.c
Comment (nicholasberlin) [inline btfhub.c:1]: What's this?

Comment (nicholasberlin) [reply btfhub.c:1]: Oh, this is adding the new offsets.

**Review by nicholasberlin — APPROVED** [review]
LGTM
```
$ git checkout ns
Branch 'ns' set up to track remote branch 'ns' from 'origin'.
Switched to a new branch 'ns'
$ make clean
  CLEAN    
$ make test
  CC       man-embedder
  MAN-EMB  quark-btf.8
  MAN-EMB  quark-mon.8
  MAN-EMB  quark-test.8
  MAN-EMB  display_man.c
  CLANG    bpf_prog.tmp.o
  BPFTOOL  bpf_prog.o
  BPFTOOL  bpf_prog_skel.h
  CC       bpf_queue.o
  CC       btf.o
  CC       btfhub.o
  CC       compat.o
  CC       kprobe_queue.o
  CC       quark.o
  CC       qutil.o
  AR       libquark.a
  AR       libquark_big.a
  CC       quark-test
sudo ./quark-test
t_probe @ ebpf...............ok
t_probe @ kprobe.............ok
t_fork_exec_exit @ ebpf......ok
t_fork_exec_exit @ kprobe....ok
t_namespace @ ebpf...........ok
t_namespace @ kprobe.........ok
t_cache_grace @ ebpf.........ok
t_cache_grace @ kprobe.......ok
t_min_agg @ ebpf.............ok
t_min_agg @ kprobe...........ok
t_stats @ ebpf...............ok
t_stats @ kprobe.............ok
0 failures
```

**Comment by haesbaert** [comment]
> LGTM
> 
> ```
> $ git checkout ns
> Branch 'ns' set up to track remote branch 'ns' from 'origin'.
> Switched to a new branch 'ns'
> $ make clean
>   CLEAN    
> $ make test
>   CC       man-embedder
>   MAN-EMB  quark-btf.8
>   MAN-EMB  quark-mon.8
>   MAN-EMB  quark-test.8
>   MAN-EMB  display_man.c
>   CLANG    bpf_prog.tmp.o
>   BPFTOOL  bpf_prog.o
>   BPFTOOL  bpf_prog_skel.h
>   CC       bpf_queue.o
>   CC       btf.o
>   CC       btfhub.o
>   CC       compat.o
>   CC       kprobe_queue.o
>   CC       quark.o
>   CC       qutil.o
>   AR       libquark.a
>   AR       libquark_big.a
>   CC       quark-test
> sudo ./quark-test
> t_probe @ ebpf...............ok
> t_probe @ kprobe.............ok
> t_fork_exec_exit @ ebpf......ok
> t_fork_exec_exit @ kprobe....ok
> t_namespace @ ebpf...........ok
> t_namespace @ kprobe.........ok
> t_cache_grace @ ebpf.........ok
> t_cache_grace @ kprobe.......ok
> t_min_agg @ ebpf.............ok
> t_min_agg @ kprobe...........ok
> t_stats @ ebpf...............ok
> t_stats @ kprobe.............ok
> 0 failures
> ```

thanks for running it, I appreciate it :D

**Pull request merged**